commercial-paper/README.md

```diff
@@ -273,9 +609,38 @@ node redeem.js
 java -cp target/commercial-paper-0.0.1-SNAPSHOT.jar org.digibank.Redeem
 ```
 
+</p>
+</details>
+
+
+**<details><summary>Perform Queries: Ownership, Asset History etc</summary>**
```

Comment:
Unless you plan to add support for this in all languages you need to say that this is only supported by the Javascript smart contract.